server/src/data/users.ts

```diff
@@ -0,0 +1,18 @@
+import * as firestore from 'firebase/firestore';
+const db = require('../firestore/config').db;
```

Comment:
I think this needs to be ../firebase/config because I get an error when I try to run it with /firestore.